### PR TITLE
Sample json

### PR DESCRIPTION
**Pull request by MikeTheCanuck**
Last night was the third time the sample JSON question came up.  Seems like a good idea to document the kind of data emitted by the endpoints, so I sampled from the info that Hassan put in Slack last week.

As he noted, this will have to occasionally be updated as the data models change, but I still believe it's worth the effort.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,22 @@ http://127.0.0.1:8000/kpm
...
 - summary: uses query parameters to return subsets of the budget data given by the 'ocrb' endpoint.
-- kpm: provides data from City of Portland Budget in Brief documents (e.g. FY 2016-17), all Service Area sections, table "Key Performance Measures".
+
+# Sample Endpoint data
+- KPM:
+```json
+[
+    {"source_document":"FY 2015-16 Budget in Brief","service_area":"Public Safety","bureau":"Bureau of Emergency Communications","key_performance_measures":"BOEC - % of priority medical calls dispatched within 90 seconds","fy":"2013-14","budget_type":"Actual ","amount":72.0,"units":"%"},{"source_document":"FY 2015-16 Budget in Brief","service_area":"Public Safety","bureau":"Bureau of Fire & Police Disability & Retirement","key_performance_measures":"FPDR - Tax levy rate per $1,000 of Real Market Value","fy":"2013-14","budget_type":"Actual ","amount":1.62,"units":"$"},    {"source_document":"FY 2015-16 Budget in Brief","service_area":"Transportation & Parking","bureau":"Portland Bureau of Transportation","key_performance_measures":"BOT – % of bridges in non-distressed condition","fy":"2013-14","budget_type":"Actual ","amount":null,"units":""},
```
Comment (hassanshamim) [inline README.md]: Could you update this with the following?  I pretty printed it, which isn't totally necessary but if there's a comma after the last item of a collection it isn't valid json.  Python is cool with it but JSON isn't.

```json
[
  {
    "source_document": "FY 2015-16 Budget in Brief",
    "service_area": "Public Safety",
    "bureau": "Bureau of Emergency Communications",
    "key_performance_measures": "BOEC - % of priority medical calls dispatched within 90 seconds",
    "fy": "2013-14",
    "budget_type": "Actual ",
    "amount": 72,
    "units": "%"
  },
  {
    "source_document": "FY 2015-16 Budget in Brief",
    "service_area": "Public Safety",
    "bureau": "Bureau of Fire & Police Disability & Retirement",
    "key_performance_measures": "FPDR - Tax levy rate per $1,000 of Real Market Value",
    "fy": "2013-14",
    "budget_type": "Actual ",
    "amount": 1.62,
    "units": "$"
  },
  {
    "source_document": "FY 2015-16 Budget in Brief",
    "service_area": "Transportation & Parking",
    "bureau": "Portland Bureau of Transportation",
    "key_performance_measures": "BOT – % of bridges in non-distressed condition",
    "fy": "2013-14",
    "budget_type": "Actual ",
    "amount": null,
    "units": ""
  }
]
```

File: README.md [hunk 2]
```diff
@@ -65,6 +65,22 @@ http://127.0.0.1:8000/kpm
...
+]
+- OCRB: 
+```json
```
Comment (hassanshamim) [inline README.md]: Same as above.

```json
[
  {
    "source_document": "FY 2015-16 Budget in Brief",
    "service_area": "Public Safety",
    "bureau": "Bureau of Emergency Communications",
    "budget_category": "Capital",
    "amount": 0,
    "fy": "2013-14",
    "budget_type": "Actual"
  },
  {
    "source_document": "FY 2015-16 Budget in Brief",
    "service_area": "Public Safety",
    "bureau": "Bureau of Fire & Police Disability & Retirement",
    "budget_category": "Capital",
    "amount": 232658,
    "fy": "2013-14",
    "budget_type": "Actual"
  },
  {
    "source_document": "FY 2015-16 Budget in Brief",
    "service_area": "Public Safety",
    "bureau": "Bureau of Emergency Communications",
    "budget_category": "Operating",
    "amount": 23346735,
    "fy": "2013-14",
    "budget_type": "Actual"
  },
  {
    "source_document": "FY 2015-16 Budget in Brief",
    "service_area": "Public Safety",
    "bureau": "Bureau of Fire & Police Disability & Retirement",
    "budget_category": "Operating",
    "amount": 162156833,
    "fy": "2013-14",
    "budget_type": "Actual"
  }
]
```

**Comment by MikeTheCanuck** [comment]
Heh, good catch. I have been well and truly Pythonized, leaving the trailing comma there on purpose ;)

Nice tips.  I'll check this when I get. Ack in front of my computer.

**Comment by MikeTheCanuck** [comment]
@hassanshamim - I haven't taken the time to forensically determine how we arrived at two requirements.txt, and I was hoping someone would have a confident idea which is the "appropriate" one, given the folder structure and where other files have landed.  If you have a suggestion for which to delete, PR that change and I'll test the Docker build with the change.  Thanks!

**Comment by MikeTheCanuck** [comment]
@hassanshamim - I'm still not sure why originally we ended up with a **/budget_proj/requirements.txt** as well as **/requirements.txt** but according to my findings during the work to stabilize #47 we definitely require **/budget_proj/requirements.txt** to enable the Docker build.

**Pull request merged**